src/test/kotlin/no/nav/familie/ef/søknad/mock/InnsynTestConfiguration.kt

```diff
@@ -0,0 +1,179 @@
+package no.nav.familie.ef.søknad.mock
+
+import no.nav.familie.ef.søknad.integration.TpsInnsynServiceClient
+import no.nav.familie.ef.søknad.integration.dto.*
+import org.mockito.Mockito.`when`
+import org.mockito.Mockito.mock
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Primary
+import org.springframework.context.annotation.Profile
+import java.time.LocalDate
+
+@Configuration
+class InnsynTestConfiguration {
+
+// eksempeldata fra TPS innsyn swagger
```

Comment:
Hvis vi har behov for dette så kanskje det kan ligge i en egen fil? :D

Reply:
Etter en runde med @Bent gikk vi for en annen løsning: Vi mocker endepunktet til tps-innsyn, ikke vår service. Da får vi med oss hele kjeden av uthenting av data og validering av respons mm. . Se ny MockController: TpsInnsynMockController